snapcraft.yaml

```diff
@@ -32,10 +32,9 @@ parts:
     source-type: git
     source-tag: 3.38.2
     plugin: meson
-    meson-parameters: [--prefix=/usr]
+    meson-parameters:
+      - --prefix=/usr
     parse-info: [usr/share/metainfo/org.gnome.Nibbles.appdata.xml]
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
-    organize:
-      snap/gnome-nibbles/current/usr: usr
+      craftctl default
+      patch -p1 < $CRAFT_PROJECT_DIR/patches/patch1.diff
```

Comment:
Could we rename the patch to something more intuitive than patch1? Also Maybe this line could be to load *.diff, in case there are future patches, wdyt?

Reply:
But does the *.diff statement preserve the order? I mean: if we call the patches 01-xxxx.diff, 02-yyy.diff... is it enough to guarantee that they are applied in alphabetical order, or they are read in the order in which they are stored in the disk's directory entry?

Reply:
Confirmed, it is.

Reply:
yeah as long as the numerical order "01-whatever.diff", "02-another-patch.diff", etc is the right order, then it will be preserved.